src/chain/mod.rs

```diff
@@ -16,6 +17,20 @@ use tokio::io::{AsyncWrite, AsyncWriteExt, BufWriter};
 
 pub use self::{store::*, weight::*};
 
+pub async fn export_from_head<D: Digest>(
+    db: Arc<impl Blockstore + SettingsStore + Send + Sync + 'static>,
+    lookup_depth: ChainEpochDelta,
+    writer: impl AsyncWrite + Unpin,
+    seen: CidHashSet,
+    skip_checksum: bool,
+) -> anyhow::Result<(Tipset, Option<digest::Output<D>>), Error> {
+    let head_key = SettingsStoreExt::read_obj::<TipsetKey>(&db, crate::db::setting_keys::HEAD_KEY)?
+        .context("chain head key not found")?;
+    let head_ts = Tipset::load(&db, &head_key)?.context("chain head not found")?;
```

Comment:
We could simplify it with `load_required`. One `?` less!

Reply:
Done.